.golangci.yaml

```diff
@@ -9,6 +9,7 @@ linters:
     - depguard
     - dupl
     - errcheck
+    - exhaustive
```

Comment:
I'm a tad bit ambivalent about this value of this one but I won't object.

Reply:
Same set of arguments in-favor/against using gcc '-Wswitch-enum' in C/C++ (which indeed, is not enabled automatically by '-Wall')